spec/support/fixtures/cli/generators/error/error_controller_fixture.cr

```diff
@@ -0,0 +1,27 @@
+module CLIFixtures
+  module ErrorControllerFixture
```

Comment:
Is this name redundant - based on the namespacing?

Reply:
I like modules to follow the file structure. What about something like `Fixtures::CLI::Generate`?

Reply:
I like that convention.

I was following what was done here: https://github.com/amberframework/amber/pull/344/files#diff-38c1446038d633c0ece1152b8988e578R1

Maybe a different PR to consistently change the naming everywhere.

Reply:
`Fixtures::CLI::Generate` 👍